app/services/batched_remove_status_service.rb

```diff
@@ -35,7 +35,7 @@ def call(statuses, **options)
 
     # Since we skipped all callbacks, we also need to manually
     # deindex the statuses
-    Chewy.strategy.current.update(StatusesIndex, statuses_and_reblogs) if Chewy.enabled?
+    Chewy::Index.update([StatusesIndex, PublicStatusesIndex], statuses_and_reblogs) if Chewy.enabled?
```

Comment:
Actually, `Chewy::Index.update` does not seem to exist. What was the intended change here?